CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 4.1.6 | [PR#4449](https://github.com/bbc/psammead/pull/4449) Add deprecation notice to psammead-amp-geo |
```

Comment:
I just merged in some changes so that the root CHANGELOG is not required

Reply:
Awesome, thanks @jroebu14!